es/03_2_Conociendo_Su_Configuracion_Bitcoin.md

```diff
@@ -0,0 +1,315 @@
+# 3.2: Conociendo la configuración de Bitcoin
+
+Antes de comenzar a jugar con Bitcoin, siempre desearía tener una mejor compresión de su configuración.
+
+## Conozca su directorio de Bitcoin
+
+Para empezar, debe comprender dónde se guarda todo: el directorio `~/.bitcoin`.
+
+El directorio principal solo contiene su archivo de configuración y el directorio testnet:
+```
+$ ls ~/.bitcoin
+bitcoin.conf  testnet3
+```
+Las guías de configuración del [Capítulo dos: Creación de un VPS de Bitcoin-Core] presentaron un archivo de configuración estandarizado. [§3.1: Verificación de la configuración de Bitcoin] sugirió cómo cambiarla para admitir configuraciones más avanzadas. Si está interesado en aprender aún más sobre el archivo de configuración, puede consultar el Generador de configuración de [Bitcoin Core de Jameson Lopp](https://jlopp.github.io/bitcoin-core-config-generator/).
```

Comment:
- Add link: `[Capítulo dos: Creación de un VPS de Bitcoin-Core](02_0_Configurando_un_Bitcoin-Core_VPS.md)`.
- Add link: `[3.1: Verificando Su Configuración Bitcoin](03_1_Verificando_Su_Configuracion_Bitcoin.md)`.
- Reference to #377